types/v56/nsxt_types.go

```diff
@@ -1106,3 +1106,101 @@ type NsxtAlbVirtualServiceApplicationProfile struct {
 	// * L4 TLS (certificate reference is mandatory)
 	Type string `json:"type"`
 }
+
+// DistributedFirewallRule represents a single Distributed firewall rule
```

Comment:
Nit, keeping the style of your other comments below:

```suggestion
// DistributedFirewallRule represents a single Distributed Firewall rule
```